uint64 utime
uint8 attempted
uint8 in_fault
RomanFaultState fault_status
